src/layouts/sidebar-sidecar/utils/_local_platform-docs-mdx/index.jsx

```diff
@@ -96,17 +96,11 @@ function EnterpriseAlert(props) {
   return <EnterpriseAlertBase product={currentProduct?.slug} {...props} />
 }
 
-// Tabs is a general-purpose component that we format for ease of use within mdx
-// It is also wrapped in a span with a css module class for styling overrides
-function Tabs({ defaultTabIdx, children }) {
-  if (!Array.isArray(children))
-    throw new Error('Multiple <Tab> elements required')
-
-  return (
-    <span className={s.tabsRoot}>
-      <TabsBase className="g-tabs" defaultTabIdx={defaultTabIdx}>
-        {children}
-      </TabsBase>
-    </span>
-  )
+/**
+ * Because the `defaultTabIdx` prop is being renamed to `initialActiveIndex`,
+ * this wrapper ensures any use of `defaultTabIdx` is still supported in older
+ * versions of docs.
+ */
```

Comment:
I'd like to rename the prop because `initialActiveIndex` is a little more clear that the index is only used on the initial load. The `default` portion of `defaultTabIdx` doesn't convey that same meaning, and I also want to remove the `Idx` abbreviation.